### ISSUE TITLE
Feature Request: Please provide `do not submit` automation bot

### ISSUE THREAD

**Issue by msampathkumar** [terraform-google-modules/terraform-docs-samples#487]
### TL;DR

For [this](https://github.com/terraform-google-modules/terraform-docs-samples) repo, enable `Do Not Submit` bot

More details @ https://github.com/googleapis/repo-automation-bots

### Terraform Resources

```markdown
NA
```

### Detailed design

```markdown
General steps
1. Go to https://github.com/apps/do-not-merge-gcf
2. Select https://github.com/terraform-google-modules/terraform-docs-samples
3. Save/Enable
```

### Additional information

[`terraform-docs-samples`](https://github.com/terraform-google-modules/terraform-docs-samples) provides the PR reviewers with `auto-merge` option for PRs. This enabled automation to merge the PRs once all the CICD tests ran successful. This is very convenient when all the CICD tests are passed but there new changes to `main` branch.

Why `do-no-merge` bot?
As seen in https://github.com/terraform-google-modules/terraform-docs-samples/pull/484, sometimes PRs need to be on hold tills some CLs or other updates are passed. Sometimes, this may be over 1-2 weeks.

As PR reviewers works rotation, providing `do-not-merge` automation help us prevent any accidental auto-merge.

**Comment by msampathkumar** [comment]
Recently there were a lot of changes and updates in CICD. The new faster CICD does not need a lot of time and PR process is simpler.
